roles/selfsigned_certs/tasks/main.yml

```diff
@@ -0,0 +1,58 @@
+---
+- name: Install pip
+  yum:
+    name: "python2-pip"
+    state: "installed"
+  when: "'loadbalancer-vm' in group_names"
+
+- name: Install python crypto module
+  pip:
+    name: "cryptography"
+    state: "latest"
+  when: "'loadbalancer-vm' in group_names"
+
+- name: Generate a private key
+  openssl_privatekey:
+    path: "{{ tls.cert_private_path }}/{{ tls_star_cert_key }}"
+    size: 2048
+  when: "'loadbalancer-vm' in group_names"
+
+- name: Generate a CSR
+  openssl_csr:
+    path: /etc/pki/tls/certs/openconext.csr
+    privatekey_path: "{{ tls.cert_private_path }}/{{ tls_star_cert_key }}"
+    common_name: "*.{{ base_domain }}"
+    organization_name: OpenConext
+    basic_constraints: CA:TRUE
```

Comment:
Yes it is unfortunately. Otherwise the certificate cannot be imported in the certificate store using the method of placing it in /etc/pki/ca-trust/source/anchors/ and running update-ca-trust